lifecycle-operator/controllers/common/test_utils.go

```diff
@@ -5,11 +5,37 @@ import (
 
```

Comment:
I would put this under a test package name, something like `testcommon`. Otherwise, this setup could lead to sneakily importing test code via `common github.com/keptn/lifecycle-toolkit/lifecycle-operator/controllers/common`